app.py

```diff
@@ -49,3 +66,93 @@ def serve_angular(path):
         ])
         return flask.redirect(target)
     return flask.send_file('dist/client/index.html')
+
+@app.route('/login')
+def login():
+    """'Login' page for auth flow"""
+    return '<a href="/auth">Authorize</a>'
+
+@app.route('/auth')
+def oauth_flow():
+    """Redirect for user authorization"""
+    genius = OAuth2Service(
+        client_id=CLIENT_ID,
+        client_secret=CLIENT_SECRET,
+        name='genius',
+        authorize_url=AUTHORIZE_URL,
+        access_token_url=ACCESS_TOKEN_URL,
+        base_url=BASE_URL)
+
+    redirect_uri = REDIRECT_URI
+
+    # TODO Define a better state
+    state = 'state'
+
+    params = {
+        'scope': 'me',
+        'response_type': 'code',
+        'redirect_uri': redirect_uri,
+        'state': state}
```

Comment:
nit: add comma

https://google.github.io/styleguide/pyguide.html#341-trailing-commas-in-sequences-of-items

```suggestion
        'state': state,
    }
```

Reply:
(here and elsewhere)